core/ir/aarch64/instr_create_api.h

```diff
@@ -658,13 +658,19 @@
     instr_create_0dst_3src((dc), OP_tbnz, (pc), (reg), (imm))
 #define INSTR_CREATE_cmp(dc, rn, rm_or_imm) \
     INSTR_CREATE_subs(dc, OPND_CREATE_ZR(rn), rn, rm_or_imm)
-#define INSTR_CREATE_eor(dc, d, s)                                      \
-    INSTR_CREATE_eor_shift(dc, d, d, s, OPND_CREATE_INT8(DR_SHIFT_LSL), \
-                           OPND_CREATE_INT8(0))
+#define INSTR_CREATE_eor(dc, d, s_or_imm)                                            \
```

Comment:
Thank you for contributing.

Ideally, every new addition of an `INSTR_CREATE_` or `XINSTR_CREATE_` should include the Doxygen comment block describing the macro, e.g.
```
/**                                                                       
 * Creates an EOR instruction with one output and two inputs.             
 * \param dc   The void * dcontext used to allocate memory for the instr_t.
 * \param rd   The output register.                                       
 * \param rn   The first input register.                                  
 * \param rm_or_imm   The second input register or immediate.             
 */
```

Reply:
Thanks, added